pkg/common/si_helper.go

```diff
@@ -133,18 +134,23 @@ func CreateReleaseAllocationRequestForTask(appID, allocUUID, partition, terminat
 	return result
 }
 
-func CreateUpdateRequestForNewNode(node Node) si.NodeRequest {
+func CreateUpdateRequestForNewNode(nodeID string, capacity *si.Resource, occupied *si.Resource,
```

Comment:
Add comment to the exported function:
```
// CreateUpdateRequestForNewNode builds a NodeRequest ....
```
Should do the same for all si_helper functions changed here.